docs/tutorial/acquiring_data/hek.rst

```diff
@@ -231,44 +227,30 @@ and as a check:
 .. code-block:: python
 
    >>> result["hek"]["fl_peakflux"] # doctest: +REMOTE_DATA
-   <QueryResponseColumn name='fl_peakflux' dtype='object' length=17>
-      None
-      None
-      None
-      None
-      None
-      None
-      None
-   2326.86
-   1698.83
-      None
-      None
-   2360.49
-   3242.64
-   1375.93
-   6275.98
-   923.984
-   1019.83
-
-   >>> result["hek"]["event_coord1"] # doctest: +REMOTE_DATA
-   <QueryResponseColumn name='event_coord1' dtype='float64' length=17>
-    51.0
-    51.0
-    51.0
-   924.0
-   924.0
-   924.0
-    69.0
-   883.2
-   883.2
-    69.0
-    69.0
-   883.2
-   883.2
-   883.2
-   883.2
-   883.2
-   883.2
+   <MaskedQuantity [     ———,      ———,      ———,      ———,      ———,
+                         ———,      ———, 2326.86 , 1698.83 ,      ———,
+                         ———, 2360.49 , 3242.64 , 1375.93 , 6275.98 ,
+                     923.984, 1019.83 ] DN / (pix s)>
+
+   >>> result["hek"]["event_coord"] # doctest: +REMOTE_DATA
+   <QueryResponseColumn name='event_coord' dtype='object' length=17>
+   <SkyCoord (Helioprojective: obstime=2011-08-08T01:30:04.000, rsun=695700.0 km, observer=<HeliographicStonyhurst Coordinate (obstime=2011-08-08T01:30:04.000, rsun=695700.0 km): (lon, lat, radius) in (deg, deg, AU)\n    (0., 6.23019163, 1.01409307)>): (Tx, Ty) in arcsec\n    (51., 151.)>
+   <SkyCoord (Helioprojective: obstime=2011-08-08T01:30:04.000, rsun=695700.0 km, observer=<HeliographicStonyhurst Coordinate (obstime=2011-08-08T01:30:04.000, rsun=695700.0 km): (lon, lat, radius) in (deg, deg, AU)\n    (0., 6.23019163, 1.01409307)>): (Tx, Ty) in arcsec\n    (51., 151.)>
+   <SkyCoord (Helioprojective: obstime=2011-08-08T01:30:04.000, rsun=695700.0 km, observer=<HeliographicStonyhurst Coordinate (obstime=2011-08-08T01:30:04.000, rsun=695700.0 km): (lon, lat, radius) in (deg, deg, AU)\n    (0., 6.23019163, 1.01409307)>): (Tx, Ty) in arcsec\n    (51., 151.)>
+   <SkyCoord (Helioprojective: obstime=2011-08-09T02:30:04.000, rsun=695700.0 km, observer=<HeliographicStonyhurst Coordinate (obstime=2011-08-09T02:30:04.000, rsun=695700.0 km): (lon, lat, radius) in (deg, deg, AU)\n    (0., 6.29395015, 1.01392568)>): (Tx, Ty) in arcsec\n    (924., 217.)>
+   <SkyCoord (Helioprojective: obstime=2011-08-09T02:30:04.000, rsun=695700.0 km, observer=<HeliographicStonyhurst Coordinate (obstime=2011-08-09T02:30:04.000, rsun=695700.0 km): (lon, lat, radius) in (deg, deg, AU)\n    (0., 6.29395015, 1.01392568)>): (Tx, Ty) in arcsec\n    (924., 217.)>
+   <SkyCoord (Helioprojective: obstime=2011-08-09T02:30:04.000, rsun=695700.0 km, observer=<HeliographicStonyhurst Coordinate (obstime=2011-08-09T02:30:04.000, rsun=695700.0 km): (lon, lat, radius) in (deg, deg, AU)\n    (0., 6.29395015, 1.01392568)>): (Tx, Ty) in arcsec\n    (924., 217.)>
+   <SkyCoord (HeliographicStonyhurst: obstime=2011-08-09T07:19:00.000, rsun=695700.0 km): (lon, lat) in deg\n    (69., 15.)>
```

Comment:
Arghl, why are some of these rows in a different frame :nauseated_face:

Reply:
🤷‍♂️